### PR TITLE
BeautifulSoup4 article added

### PR DESCRIPTION
**Pull request by Aman-Clement**

### CODE REVIEW

**Review by ayushjain01 — CHANGES_REQUESTED** [review]
Good to go.
Image file names are inconsistent

File: _data/authors.yml [hunk 1]
```diff
@@ -6,4 +6,9 @@ ashok:
 ayush:
   picture: # replace with avatar later
   name: Ayush Jain
-  twitter: AyushGanna
\ No newline at end of file
+  twitter: AyushGanna
+
```
Comment (tankala) [inline _data/authors.yml:10]: now authors.yml not required

File: _data/authors.yml [hunk 2]
```diff
@@ -6,4 +6,9 @@ ashok:
...
+aman:
```
Comment (ayushjain01) [inline _data/authors.yml]: No need to add in authors.yml anymore, you can add directly in the front matter

**Review by ayushjain01 — CHANGES_REQUESTED** [review]

File: _gettingstarted/beautifulsoup4.md [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+tags: [python, pip, piptrends, beautifulsoup, beautifulsoup4, bs4]
+image: https://learn.piptrends.com/assets/images/getting-started/getting-started-beautifulsoup4-cover.png
+cover: /assets/images/getting-started/beautifulsoup4.png
+article_header:
+  type: overlay
+  theme: dark
+  background_image:
+    gradient: 'linear-gradient(135deg, #5200FF, #FF8A98)'
+date: 2023-12-14
+aman:
```
Comment (ayushjain01) [inline _gettingstarted/beautifulsoup4.md]: Change this to author

**Review by ayushjain01 — APPROVED** [review]

**Review by ayushjain01 — APPROVED** [review]

**Review by tankala — APPROVED** [review]

**Pull request merged**